templates/common/on-prem/files/keepalived.yaml

```diff
@@ -100,8 +100,8 @@ contents:
           remove_vip()
           {
             address=$1
-            interface=$(ip -o a | grep '\s${address}/' | awk '{print $2}')
-            cidr=$(ip -o a | grep '\s${address}/' | awk '{print $4}')
+            interface=$(ip -o a | awk "/\s${address}/ {print \$2}")
```

Comment:
This doesn't do quite the same thing. The awk needs to be:

awk "/\s${address}\// {print \$2}"

We need to make sure we match exactly the VIP and not a substring.